docs/ref-datastore-manipulating-data.md

```diff
@@ -47,11 +41,7 @@ Currently supported operators(more coming soon)
 You can also create predicate expressions
 
 ```typescript
-query({ 
-    title: "",
-    numberOfDaysLeft: ""
-    __typename: "Task"
-}, (p: any) => p.or(
+TaskModel.query((p) => p.or(
     p.title("eq", "test"),
```

Comment:
EQ etc. might be constants on Filter object. Worth exploring later

Reply:
Aye. It won't be difficult to change to constants